### PR TITLE
Updating QA Setup Script to not explicitly set remote branch for build.sh

### PR DESCRIPTION
**Pull request by ryanrath**
## Description
I've updated the `qa-test-setup` script  to not explicitly set a remote branch for `build.sh`. This was causing problems because it was using the `BASE_BRANCH` env variable which in certain situations would be empty. This ultimately had `build.sh` erroring out w/ the message: `Could not resolve remote branch: `. 

## Motivation and Context
It's a good thing to have our QA tests actually run.

## Tests performed
I've tested this change locally w/ both a base XDMoD & SUPREMM build using the last merged PR's for both as the target. Both builds complete successfully.

## Checklist:

- [X] The pull request description is suitable for a Changelog entry
- [X] The milestone is set correctly on the pull request
- [X] The appropriate labels have been added to the pull request

### CODE REVIEW

File: tests/ci/scripts/qa-test-setup.sh [hunk 1]
```diff
@@ -24,10 +24,10 @@ if [[ "$XDMOD_TEST_MODE" == "upgrade" ]]; then
     # Setup the xdmod-qa environment / requirements.
     $HOME/.qa/scripts/install.sh
 
-    # If we're running on Shippable then make sure to include the
-    # base branch that we're merging into.
+    # If we're running on Shippable and a base branch is present then explicitly specify
+    # it as the remote branch for build.sh.
```
Comment (jpwhite4) [inline tests/ci/scripts/qa-test-setup.sh:28]: Under what circumstances would you be running on shippable and a base branch would not be present? What would happen to the build in this case?

Comment (ryanrath) [reply tests/ci/scripts/qa-test-setup.sh:28]: These seem to be the conditions:
https://app.shippable.com/github/ubccr/xdmod/runs/6422/2/scripts
https://app.shippable.com/github/ubccr/xdmod-supremm/runs/1586/2/scripts

*Expand `reports` and search for `BASE_BRANCH`*

And it looks like the build fails.

Comment (jtpalmer) [reply tests/ci/scripts/qa-test-setup.sh:28]: When is the base branch ever set?  I don't see it anywhere else.

Comment (ryanrath) [reply tests/ci/scripts/qa-test-setup.sh:28]: It's a env variable that Shippable provides, sometimes.

**Comment by ryanrath** [comment]
Just submitted a PR https://github.com/ubccr/xdmod-qa/pull/19 that should resolve the problems w/ this PR.

**Review by jpwhite4 — APPROVED** [review]

**Pull request merged**